contracts/Users.sol

```diff
@@ -118,6 +122,17 @@ contract Users {
         return coll;
     }
 
+    function setUsers(uint newLimit) onlyOwner returns (bool) {
+        if(newLimit <=  maxLimit)
```

Comment:
so, no decrease allowed there

Reply:
yes, as you mentioned you will be needing  to increase the count to add new Users